package.json

```diff
@@ -14,7 +14,7 @@
         "ts-node": "7.0.1",
         "tslint": "^5.11.0",
         "tslint-language-service": "^0.9.9",
-        "typescript": "^3.0.1"
+        "typescript": "^3.9.0"
     },
     "scripts": {
         "prepare": "lerna run clean && yarn build && yarn bundle",
```

Comment:
Actually, I see an issue here locally - build runs lint and compile in parallel, but the example file depends on a compiled file in the server.

```
$ tslint -c ../tslint.json --project .
[compile] example/src/client.ts(13,41): error TS2307: Cannot find module 'typescript-language-server/lib/ts-protocol' or its corresponding type declarations.
[compile] example/src/client.ts(61,36): error TS2345: Argument of type 'null' is not assignable to parameter of type 'IPosition'.
[compile] example/src/client.ts(61,58): error TS2339: Property 'position' does not exist on type 'CancellationToken'.
```

Reply:
Looks like this was just because I had old/invalid .tsbuildinfo files

Reply:
https://github.com/theia-ide/typescript-language-server/pull/169